### DISCUSSION TITLE
Discord Server?

### DISCUSSION THREAD

**Discussion by nailuj29** [bevyengine/bevy#1090] [General]
Is there a Discord server for Bevy? If not, I think it would be a great idea to make one.

**Comment by mockersf** [comment]
There is one, you can find it in the [Community section](https://github.com/bevyengine/bevy#community). Come and chat!

**Reply by ahkohd** [reply]
The discord invite link is expired.

**Reply by mockersf** [reply]
Link to https://discord.gg/bevy is still working for me?
